esp/esp/dbmail/models.py

```diff
@@ -218,6 +224,7 @@ def createRequest(var_dict = None, *args, **kwargs):
 
         if var_dict is not None:
             new_request.save()
+            var_dict['request'] = new_request
```

Comment:
What is this new line for?

Reply:
This adds the message request as a variable for the email rendering. I added a new tag to the comm panel (`{{ request.public_url }}`) that utilizes this variable.